server/configuration.go

```diff
@@ -18,6 +18,13 @@ import (
 // If you add non-reference types to your configuration struct, be sure to rewrite Clone as a deep
 // copy appropriate for your types.
 type configuration struct {
+	TenantId      string
+	ClientId      string
+	ClientSecret  string
+	BotUsername   string
+	BotPassword   string
+	WebhookSecret string
+	EnabledTeams  string
```

Comment:
It seems off that these have `camelCase` JSON keys in `plugin.json`, but this casing is `PascalCase`. Is this automatically handled by Go's json parser?

Reply:
Yes, that is handled by the go json parser. I can make it consistent if you prefer it.